.github/workflows/docker.yml

```diff
@@ -26,21 +17,7 @@ jobs:
         with:
           persist-credentials: false
         uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
-      - name: Build and push - Probe
-        uses: grafana/shared-workflows/actions/build-push-to-dockerhub@402975d84dd3fac9ba690f994f412d0ee2f51cf4 # build-push-to-dockerhub-v0.1.1
-        with:
-          repository: ${{ env.REGISTRY_IMAGE }}
-          context: .
-          push: true
-          file: Dockerfile-probe
-          platforms: linux/amd64
-          tags: ${{ env.TAGS_CONFIG }}
-      - name: Build and push - Runner
-        uses: grafana/shared-workflows/actions/build-push-to-dockerhub@402975d84dd3fac9ba690f994f412d0ee2f51cf4 # build-push-to-dockerhub-v0.1.1
-        with:
-          repository: ${{ env.REGISTRY_IMAGE }}
-          context: .
-          push: true
-          file: Dockerfile-runner
-          platforms: linux/amd64
-          tags: ${{ env.TAGS_CONFIG }}
+      - name: Dockerhub Login
+        uses: grafana/shared-workflows/actions/dockerhub-login@49c90b10fcbce463983bed45932cf468b8bd06ce # dockerhub-login latest
+      - name: Build and push
+        run: make docker-build docker-push
```

Comment:
How about splitting this into two steps, one for building and the other for pushing? That way might be easier to test locally each step separately.

Reply:
Oh good call, I'll do that in a follow up because I just wanna get images into the repo! ☺️